pyanaconda/bootloader/base.py

```diff
@@ -61,16 +62,27 @@ def _get_iscsi_node_from_device(device):
 
 def _is_on_ibft(device):
     """Tells whether a given device is ibft disk or not."""
-    iscsi_proxy = STORAGE.get_proxy(ISCSI)
     for disk in device.disks:
         if not isinstance(disk, blivet.devices.iScsiDiskDevice):
             return False
         node = _get_iscsi_node_from_device(disk)
-        if not iscsi_proxy.IsNodeFromIbft(Node.to_structure(node)):
+        if not _is_node_from_ibft(node):
             return False
     return True
 
 
+def _is_node_from_ibft(node):
+    """Is the node configured from iBFT table?.
+
+    :param node: the node information
+    """
+    for ibft_node in iscsi.ibft_nodes:
```

Comment:
Looking into blivet/iscsi.py it seems that the value of ibft_nodes is not set until iscsi.setup() is called, so maybe here it is not set?

Reply:
I guess that you mean `iscsi.startup()`. We call that during the initialization of the Storage module and this code is used only by the Storage module, so ibft_nodes should be already set when we call this function.

Reply:
Ah, it is used only by the Storage module, sorry for the noise then...